perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/TasksManagerMethod.java

```diff
@@ -112,9 +112,9 @@ public Integer call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @param task int Task
 	 * @return List<TaskResult> Results
 	 */
-	getTaskResultsForGUIByTaskOnlyNewest {
+	getTaskResultsForGUIByTaskOnlyNewestGUI {
```

Comment:
You can create all new version of these methods (like getTaskResultsByTaskOnlyNewest) and set all the old as deprecated (they will be just temporary here).

Reply:
Done. Also removed the GUI suffix from this method, this was unintended and nonsensical change.